psutil/tests/test_memory_leaks.py

```diff
@@ -288,6 +289,7 @@ def test_num_fds(self):
         self.execute(self.proc.num_fds)
 
     @skip_if_linux()
+    @unittest.skipIf(AIX, "num_ctx_switches not available on AIX")
```

Comment:
Please instead define a HAS_NUM_CTX_SWITCHES global in `tests/__init__.py` and use it here. I prefer that way so I can keep track of supported things in a single place, in case their support is added later.